notification/twilio/voice.go

```diff
@@ -531,23 +471,15 @@ func (v *Voice) ServeTest(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	var messagePrefix string
+	resp := newTwiMLResponse(w)
 	switch call.Digits {
 	default:
-		messagePrefix = "I am sorry. I didn't catch that. "
+		resp.SayUnknownDigit()
 		fallthrough
 	case "", digitRepeat:
-		message := fmt.Sprintf("%s%s. To repeat this message, press %s.", messagePrefix, call.msgBody, digitRepeat)
-		g := &gather{
-			Action:    v.callbackURL(ctx, call.Q, CallTypeTest),
-			Method:    "POST",
-			NumDigits: 1,
-			Timeout:   10,
-			Say:       message,
-		}
-		renderXML(w, req, twiMLGather{
-			Gather: g,
-		})
+		resp.Say(call.msgBody)
+		resp.AddOptions(optionStop)
```

Comment:
yeah, since test messages are disabled when unenrolled I guess if it ever had a problem where a user was being notified to give them the option to disable scheduling for their number